lang/en.json

```diff
@@ -94,7 +94,7 @@
     
     "VTM5E.Level": "Level",
     "VTM5E.Add": "Add",
-    "VTM5E.AddPower": "Add",
+    "VTM5E.AddPower": "Add Power",
```

Comment:
I believe they do, yes, since from what I see the other translations for "AddPower" are also just "add." But it's not necessary to do it for all of them I think. I just thought English would benefit from it being more specific (and I don't want to end up mis-translating another language, so it can be fixed on a case-by-case basis.)